src/SDKs/DataBox/Management.DataBox/Microsoft.Azure.Management.DataBox.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<!-- Please do not move/edit code below this line -->
+	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+	<!-- Please do not move/edit code below this line -->
+
+	<PropertyGroup>
+		<PackageId>Microsoft.Azure.Management.DataBox</PackageId>
+		<Description>Microsoft Azure Management DataBox library</Description>
+		<AssemblyName>Microsoft.Azure.Management.DataBox</AssemblyName>
+		<Version>1.0.0-preview</Version>
```

Comment:
@kushalvms if it's your first version, we recommend you start your version from 0.x.x-preview and all first preview pacakges start at 0.9.0-preview